packages/kolibri/apiSpec.js

```diff
@@ -1,5 +1,97 @@
 /*
  * This file defines the API for the core Kolibri app.
+ * This file is automatically generated using the kolibri package.json
+ * exports and exposes fields by the kolibri-tools package.
  */
 
-export default {};
+const kolibri_apiResource = require('kolibri/apiResource');
```

Comment:
I don't know why we are using the `_` here. I don't have any strong opinions about why we should or shouldn't, but just wondering what the thinking is. we have updated from the prevoius `.` (i.e. the update from kolibri.coreVue.AppBar or whatnot). But I don't know if it's about having a clear delineation in time that this has happened? or if it's just about a preference for snake case? or if it's somehow similar to the leading `__` for private functions? Or other connection to this question/comment in the reviewer guidance
>Is it obvious enough which things are private and which are public in this package

I just feel like there's something I missing here about this change, when reading it

Reply:
Hrm, I'll change how I am generating the apiSpec, I can just make the requires inline without needing the intermediary variables. You're getting tripped up on the variable name containing a `_` it seems, but that is not exposed anywhere outside of this file, it is entirely internal.